pkg/bootstrap/types.go

```diff
@@ -13,6 +13,7 @@ type KuryrBootstrapResult struct {
 	ExternalNetwork   string
 	ClusterID         string
 	OctaviaProvider   string
+	OctaviaDL         bool
```

Comment:
Even if it's effectively double listeners at the moment, I'd rather use multiple listeners. And probably it's worth to be explicit, golang doesn't have limit on line length, so "OctaviaMultipleListeners" is fine.

Reply:
Sounds good! I'll modify it